lib/npm.js

```diff
@@ -90,7 +90,12 @@ NPM.prototype.listDependencies = function listDependencies(obj, name, prefix) {
         };
     }
 
-    var fileName = name + '-' + newObj.version + '.tgz';
+    // npm pack will remove @ symbols from the name.
+    var santitizedName = name.replace(/[@]/, '');
```

Comment:
I assume there may be additional characters that will be added in the future, `@` is just the one that bit me. I can update if you'd like it, I did it this way to make it obvious that there may be other charactes that should be sanitized